resources/views/partials/topbar.twig

```diff
@@ -55,6 +38,37 @@
                                 </li>
                             </ul>
                         </li>
+                        {% set helpColor = setting_value('visiosoft.theme.defaultadmin::primary_color') %}
+                        {% if setting_value('visiosoft.theme.defaultadmin::dark_mode') %}
+                            {% set helpColor = setting_value('visiosoft.theme.defaultadmin::dark_text_color') %}
+                        {% endif %}
+                        <li class="nav-item dropdown floatright" style="padding-top: 5px;">
+                            <a href="javascript:void(0);" data-toggle="dropdown">
+                                <svg xmlns="http://www.w3.org/2000/svg" width="31.676" height="31.676" viewBox="0 0 31.676 31.676">
```

Comment:
change to img() function for svg file

Reply:
no, I need to change color in SVG dynamically. I defined a variable for color based on theme mode (light or dark), it used in the SVG

Reply:
You can call it with the `{{ img('path').data|raw }} ` and you can replace it color with css

Reply:
it is a good idea. thank you 🥇